src/test/java/edu/kis/powp/jobs2d/CompoundCommandTest.java

```diff
@@ -0,0 +1,67 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+
+import java.util.Iterator;
+
+public class CompoundCommandTest {
+    public static void main(String[] args) {
+        DrawPanelController drawController = new DrawPanelController();
+        ILine line = LineFactory.getBasicLine();
+
+        Job2dDriver driver = new LineDriverAdapter(drawController, line, "Name");
+        DriverManager driverManager = new DriverManager();
+        driverManager.setCurrentDriver(driver);
+
+        testExecuteMethod(driverManager);
+        testIteratorMethod();
+        testGetterAndSetter();
+    }
+
+    private static void testExecuteMethod(DriverManager driverManager) {
+        CompoundCommand compoundCommand = new CompoundCommand();
+        compoundCommand.addCommand(new SetPositionCommand(0, 0));
+        compoundCommand.addCommand(new OperateToCommand(50, 50));
+        compoundCommand.addCommand(new OperateToCommand(-50, 50));
+
+        compoundCommand.execute(driverManager.getCurrentDriver());
```

Comment:
The application enables executing commands loaded in the CommandManager (setCurrentCommand). You add tests either as unit tests, but primarily extend testJobs2dApp.